src/uni_modules/wot-ui/components/wd-color-picker/index.scss

```
@@ -0,0 +1,367 @@
+    transform: translate(-50%, -50%);
+  }
+
+  @include e(slider) {
+    margin-top: $color-picker-slider-margin-top;
+
+    &.wd-slider--horizontal {
+      height: $color-picker-slider-height;
+    }
+
+    .wd-slider__bar--default,
+    :deep(.wd-slider__bar--default) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)